<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CONTROLLER</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 212, 212);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="stop_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(124, 124, 124);
color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="record_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>83</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(124, 124, 124);
color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="send_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(168, 168, 168);
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="connect_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 194, 194);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="disconnect_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 194, 194);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QSlider" name="angle1_slider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="angle2_slider">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>120</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="angle3_slider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>285</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="angle4_slider">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>285</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="angle5_slider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="angle6_slider">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="output_screen">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <underline>false</underline>
      <fontweight>ExtraBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Angle Value</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="record_st">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(152, 152, 152);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>STATUS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="angle1_value">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 188, 188);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle2_value">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 188, 188);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle3_value">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>285</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 188, 188);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle4_value">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>285</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 188, 188);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle5_value">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 188, 188);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle6_value">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 188, 188);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="cnt_status">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 194, 194);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Disconnected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>541</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(104, 104, 104);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/Ảnh Ổ D/logo-hust.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>600</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/Ảnh Ổ D/hust.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nhóm 7 &lt;/p&gt;&lt;p&gt;Nguyễn Đức Huy &lt;/p&gt;&lt;p&gt;Vũ Thành Đức &lt;/p&gt;&lt;p&gt;Đỗ Hoàng Minh&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>stop_btn</zorder>
   <zorder>record_btn</zorder>
   <zorder>send_btn</zorder>
   <zorder>connect_btn</zorder>
   <zorder>disconnect_btn</zorder>
   <zorder>angle1_slider</zorder>
   <zorder>angle2_slider</zorder>
   <zorder>angle3_slider</zorder>
   <zorder>angle4_slider</zorder>
   <zorder>angle5_slider</zorder>
   <zorder>angle6_slider</zorder>
   <zorder>output_screen</zorder>
   <zorder>record_st</zorder>
   <zorder>angle1_value</zorder>
   <zorder>angle2_value</zorder>
   <zorder>angle3_value</zorder>
   <zorder>angle4_value</zorder>
   <zorder>angle5_value</zorder>
   <zorder>angle6_value</zorder>
   <zorder>cnt_status</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuController">
    <property name="title">
     <string>Nhóm 7</string>
    </property>
   </widget>
   <addaction name="menuController"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
